transformations/gender_randomizer/README.md

```diff
@@ -0,0 +1,45 @@
+# Gender Randomizer 🦎  + ⌨️ → 🐍
+
+Author name: Tabitha Sugumar
+Author email: t.sugumar@elsevier.com
+Author Affiliation: Elsevier
+
+## What type of a transformation is this?
+This transformation changes names in English texts, randomizing selection so there's an even chance of male and female names. It modifies pronouns to match the selected name.
```

Comment:
Please add an acknowledgement that names are not deterministic identifiers of someones pronouns/gender :)

Reply:
Added!